lib/puppet/provider/docker_compose/ruby.rb

```diff
@@ -0,0 +1,29 @@
+Puppet::Type.type(:docker_compose).provide(:ruby) do
+  desc "Support for Docker Compose"
+  commands :dockercompose => "docker-compose"
+    
+  mk_resource_methods
+
+
+  def exists?
```

Comment:
Thinking more about this, should this represent the presence of a file as it does now, or represent the fact the stack of containers it represents are running?